azurelinuxagent/common/utils/cryptutil.py

```diff
@@ -128,3 +132,12 @@ def bits_to_bytes(self, bits):
                 index = 7
         return bytes(byte_array)
 
+    def decrypt_secret(self, encrypted_password, private_key):
+        decoded = base64.b64decode(encrypted_password)
+        p = subprocess.Popen([self.openssl_cmd, 'cms', '-decrypt', '-inform', 'DER', '-inkey', private_key, '-in', '/dev/stdin'], stdout=subprocess.PIPE, stdin=subprocess.PIPE, stderr=subprocess.STDOUT)
```

Comment:
The args format makes this a little difficult to read/search. Consider using a const which gets split as we do elsewhere.

Reply:
agreed, it looks better.  Fixed.

---
In reply to: [194024521](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r194024521) [](ancestors = 194024521)